etc/profile-a-l/bcompare.profile

```diff
@@ -0,0 +1,69 @@
+# Firejail profile for Beyond Compare by Scooter Software
+# Description: directory and file compare utility
+# Disables the network, which only impacts checking for updates.
+# This file is overwritten after every install/update
+
+# Persistent local customizations
+include bcompare.local
+# Persistent global definitions
+include globals.local
+
+# In case the user decides to include disable-programs.inc, still allow KDE's Gwenview to view images via right click -> Open With -> Associated Application
+noblacklist ${HOME}/.config/gwenviewrc
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+# Allow comparing e.g. Firefox profile files, so don't disable access to programs
+# include disable-programs.inc
+# Uncommenting this breaks launch
+# include disable-shell.inc
+include disable-write-mnt.inc
+# Don't disable ${DOCUMENTS}, ${MUSIC}, ${PICTURES}, ${VIDEOS}
+# include disable-xdg.inc
+
+# include whitelist-common.inc
+# include whitelist-runuser-common.inc
+# include whitelist-usr-share-common.inc
+# include whitelist-var-common.inc
+
+#allusers
+apparmor
+caps.drop all
+# ipc-namespace
+machine-id
```

Comment:
Either comment this too or uncomment `nosound` (`machine-id` breaks PA).

Reply:
Launching VLC from the context menu of an MP3 does play it just fine with `machine-id` enabled, on my KDE neon 5.21. But I've commented `machine-id` anyway.

On a related note, despite `novideo`, VLC launched from the context menu of an `mp4`, does play it.

Also, should `nodvd` be uncommented? Is that a multimedia API of some sort, so if the user wants to compare files from a DVD, `nodvd` wouldn't have an effect because the DVD would be mounted via `/media/` ?

Reply:
`nodvd`: `/dev/sr0`, `/dev/sr1`, ...
`novideo`: `/dev/video0`, `/dev/video1`, ...

Reply:
> Launching VLC from the context menu of an MP3 does play it just fine with machine-id enabled, on my KDE neon 5.21. But I've commented machine-id anyway.

Second time I hear that PA works with `machine-id`. Anyway it was broken for years and until someone can confirm this for Debian/OpenSUSE Leap over Fedora to Arch/OpenSUSE Tumbleweed we should should comment it.

> On a related note, despite novideo, VLC launched from the context menu of an mp4, does play it.

Because `novideo` is about your webcam

> Also, should nodvd be uncommented? Is that a multimedia API of some sort, so if the user wants to compare files from a DVD, nodvd wouldn't have an effect because the DVD would be mounted via /media/ ?

DVD are usually accessed via gvfs/gio/kio/... which escape the sandbox via D-Bus.

Reply:
Sent a [PR re. `novideo` become about webcams](https://github.com/netblue30/firejail/pull/4100).